pkg/scheduler/cache/cache.go

```diff
@@ -1487,6 +1487,8 @@ func (sc *SchedulerCache) RecordJobStatusEvent(job *schedulingapi.JobInfo, updat
 			reason, msg, nominatedNodeName := job.TaskSchedulingReason(taskInfo.UID)
 			if len(msg) == 0 {
 				msg = baseErrorMessage
+			} else if pgUnschedulable {
```

Comment:
Seems every failed scheduling pod will add a same prefix message no matter gang plugin is enabled or not, and it's duplicated, too.